src/internals/tuples/Tuples.ts

```diff
@@ -7,6 +7,49 @@ import { Iterator, Stringifiable } from "../helpers";
 export namespace Tuples {
   type HeadImpl<xs> = xs extends readonly [infer head, ...any] ? head : never;
 
+  /**
+   * Get an element from a tuple at a given index.
+   * @param args[0] - The index of the element to get.
+   * @param args[1] - The tuple to get the element from.
+   * @param index - The index of the element to get.
+   * @param tuple - The tuple to get the element from.
+   * @returns The element at the specified index.
+   * @example
+   * ```ts
+   * type T0 = Call2<Tuples.At, 1, ["a", "b", "c"]>; // "b"
+   * type T1 = Eval<Tuples.At<1, ["a", "b", "c"]>>; // "b"
+   * type T2 = Call<Tuples.At<1>, ["a", "b", "c"]>; // "b"
+   * ```
+   */
+  export type At<
+    index extends number | _ | unset = unset,
+    tuple = unset
+  > = Functions.PartialApply<AtFn, [index, tuple]>;
+
+  interface AtFn extends Fn {
+    return: Fn.arg1<this, readonly any[]>[Fn.arg0<this, number>];
+  }
+
+  type EmptyImpl<tuple extends unknown[]> = [] extends tuple ? true : false;
+
+  interface EmptyFn extends Fn {
+    return: EmptyImpl<Fn.arg0<this, unknown[]>>;
+  }
+
+  /**
+   * Check if a tuple is empty.
+   * @param args[0] - The tuple to check.
+   * @param tuple - The tuple to check.
+   * @returns `true` if the tuple is empty, `false` otherwise.
+   * @example
+   * ```ts
+   * type T0 = Call<Tuples.Empty, []>; // true
+   * type T1 = Call<Tuples.Empty, [1, 2, 3]>; // false
+   * type T2 = Eval<Tuples.Empty<[]>>; // true
+   * ```
+   */
+  export type Empty<tuple = unset> = Functions.PartialApply<EmptyFn, [tuple]>;
```

Comment:
Could we call this IsEmpty instead? When reading `Empty` I was expecting this to return an empty tuple